packages/now-cli/src/util/dev/validate.ts

```diff
@@ -70,7 +70,9 @@ function validateKey(
 
     const error = validate.errors[0];
 
-    return `Invalid \`${key}\` property: ${error.dataPath} ${error.message}`;
+    return `Invalid \`${String(key)}\` property: ${error.dataPath} ${
```

Comment:
Seems weird… 🤔

Reply:
@TooTallNate Since I modified the type of NowConfig, not every key is a string anymore so TS was complaining.

Reply:
I see, makes sense.